src/components/pages/TodoHome.vue

```diff
@@ -22,19 +24,36 @@ import { ref, computed, toRaw } from "vue";
 import TodoHeader from "../pages/TodoHeader.vue";
 import BaseCard from "../ui/BaseCard.vue";
 import BaseSearch from "../ui/BaseSearch.vue";
+import BaseSort from "../ui/BaseSort.vue";
 import TodoWelcome from "../pages/TodoWelcome.vue";
 import { Todo } from "../../types/todo";
 
 const todos = ref<Todo[]>([]);
 const currentClickedId = ref<string | null>(null);
 const searchQuery = ref<string>("");
+const sortDirection = ref<"asc" | "desc">("asc");
+
+function getTodosFromLocalStorage() {
```

Comment:
You shouldnt have a function for this, you can add initial value to the todos right away